app/models/katello/content_view_version_import_history.rb

```diff
@@ -0,0 +1,26 @@
+module Katello
+  class ContentViewVersionImportHistory < Katello::Model
+    audited except: :metadata
+
+    belongs_to :content_view_version, :class_name => "Katello::ContentViewVersion", :inverse_of => :import_histories
+    validates_lengths_from_database
+
+    validates :content_view_version_id, presence: true
+    validates :metadata, presence: true
+    validates :path, presence: true
+    serialize :metadata, Hash
+
+    scope :with_organization_id, ->(organization_id) do
+      where(:content_view_version_id => ContentViewVersion.with_organization_id(organization_id))
+    end
+
+    scope :with_content_view_id, ->(cv_id) do
```

Comment:
why do we need this?

Reply:
I guess in the case of export_history we need these 2 scopes at https://github.com/Katello/katello/blob/master/app/controllers/katello/api/v2/content_exports_controller.rb#L24-L25

If you plan on adding similar endpoint to the imports I 'd be ok with these in the model.

Reply:
I modeled the import history model after the existing export history one, so tbh this was just left over from that :)

If I end up adding the endpoint I'll keep it in